sdk/exportentities/v2/workflow_test.go

```diff
@@ -973,6 +973,25 @@ workflow:
     - aa_2
     when:
     - manual
+`,
+		},
+		{
+			name: "Workflow with multiple integration",
+			yaml: `name: workflow-multiple-integration
+version: v2.0
+workflow:
+  test:
+    pipeline: test
+notifications:
+- type: event
+  integration: my-integration
+integrations:
+  artifactory:
+    type: artifact_manager
+    config:
+      build.info:
+        type: string
```

Comment:
is it possible to omit the type if it's string ? i remember it is possible on other "as-code integration"

Reply:
Integration does not have this kind of behaviour